website/docs/cli/commands/modules.mdx

```diff
@@ -0,0 +1,72 @@
+---
+page_title: 'Command: modules'
+description: >-
+  The terraform modules command prints source and version data about all declared
+  modules in Terraform configuration.
+---
+
+# Command: modules
+
+The `terraform modules` command provides a holistic view of all Terraform
+modules declared in Terraform configuration for the current working directory.
+This can be useful for auditing or policy purposes on module consumption. The
+output will include a list of each declared module's key, source and version.
+
+-> `terraform modules` requires **Terraform v1.10.0 or later**.
+
+## Usage
+
+Usage: `terraform modules [options]`
+
+The following flags are available:
+
+- `-json` - Displays the module declarations in a machine-readable, JSON format.
+
+Please note that, at this time, the `-json` flag is a _required_ option. In future releases, this command will be extended to allow for additional options.
+
+The output includes a `format_version` key, which as of Terraform 1.10.0 has
+value `"1.0"`. The semantics of this version are:
+
+- We will increment the minor version, e.g. `"1.1"`, for backward-compatible
+  changes or additions. Ignore any object properties with unrecognized names to
```

Comment:
```suggestion
  changes or additions. Ignore object properties with unrecognized names to
```

Reply:
How will folks know what an "unrecognized name" is? Maybe you could provide an example?